deal.II-toolchain/packages/cmake.package

```diff
@@ -1,30 +1,27 @@
-MAJOR=3.16
-MINOR=3
+MAJOR=3.20
+MINOR=5
```

Comment:
Would it make sense to use version 3.20.4 as this is also the currently used version on the ubuntu github actions runners?

Reply:
I used the lasted stable from today.